### DISCUSSION TITLE
sls uses "pipenv lock --requirements" which is deprecated

### DISCUSSION THREAD

**Discussion by eriklilljequist** [serverless/serverless#11391] [General]
Should I report this somewhere or is there some good way to manage this? Other than downgrading pipenv?

Im using:
pipenv 2022.9.9
serverless 3.22.0
serverless-python-requirements 5.4.0

Getting error: "Error: `pipenv lock --requirements --keep-outdated` Exited with code 2"

Checking pipenv release info I see "pipenv lock --requirements" has been deprecated and removed in favour of "pipenv requirements": 
- https://pipenv.pypa.io/en/latest/changelog/#id61

I dont know if this a bug in the requirements plugin or in sls itself. 

Wanted the communities thoughts before reporting a bug in the potentially wrong place.

Solved it temporarily by downgrading pipenv.

**Comment by eriklilljequist** [comment]
Answering myself: This is a known bug in serverless-python-requirements plugin.

**Reply by matt-ward** [reply]
Can you link to the issue in the plugin repo?

**Comment by pgrzesik** [comment]
Hey @eriklilljequist and @matt-ward. This should be already resolved in the latest release of the plugin. Please let me know if you still run into any issues 🙇
